src/nvim/api/buffer.c

```diff
@@ -1021,7 +1021,7 @@ void nvim_buf_del_var(Buffer buffer, String name, Error *err)
 /// @param buffer     Buffer handle, or 0 for current buffer
 /// @param[out] err   Error details, if any
 /// @return Buffer name
-String nvim_buf_get_name(Buffer buffer, Error *err)
+String nvim_buf_get_name(Buffer buffer, Arena *arena, Error *err)
```

Comment:
Is the arena argument required here?

Reply:
Yes, otherwise, the wrapper would have attempted to free the return value.

Reply:
Ah yes, sorry of course.